webanno-model/src/main/java/de/tudarmstadt/ukp/clarin/webanno/model/AnnotationFeature.java

```diff
@@ -83,6 +83,9 @@
 
     private boolean visible = true;
     
+    @Column(nullable = false)
+    private boolean featureShowHover = false;
```

Comment:
How about `includeInHover`?

Reply:
Yep, that sounds better. Changed in 02f0e68f8397f21df4c83d35c9b01113465a3edc